src/libraries/System.Text.Json/src/System/Collections/Generic/ValueListBuilder.Pop.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Collections.Generic
+{
+    /// <summary>
+    /// These public methods are required by RegexWriter.
+    /// </summary>
+    internal ref partial struct ValueListBuilder<T>
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public T Pop()
+        {
+            _pos--;
+            return _span[_pos];
+        }
+    }
```

Comment:
same comment as: https://github.com/dotnet/runtime/pull/71651/files#r914021221

Reply:
It def can, and i've made that change in https://github.com/dotnet/runtime/pull/71652.  I'll port that over to this one.